configs/config

```diff
@@ -1,7 +1,5 @@
 BR2_arm=y
 BR2_arm1176jzf_s=y
-BR2_CCACHE=y
```

Comment:
Does this do or not do anything?

Reply:
ccache was the reason why the builds didn't work anymore, see my commit message:

    fix github actions build
    
    The build failed due to a conflict with ccache and cmake,
    which arose due to a recent version upgrade of cmake in the
    Github Ubuntu image.
    
    As ccache didn't give us any value on a cold cache, and
    little value on a hot cache (all buildroot packages are
    already built and we only need to compile changed packages)
    I see little value in maintaining it.
    
    Additionally this change bumps the Buildroot version to the
    latest stable version and uses ubuntu-20.04 which will
    become ubuntu-latest anyway soon.